controls/stig_rhel9.yml

```diff
@@ -0,0 +1,4233 @@
+policy: 'Red Hat Enterprise Linux 9 Security Technical Implementation Guide'
+title: 'Red Hat Enterprise Linux 9 Security Technical Implementation Guide'
+id: stig_rhel9
+source: https://public.cyber.mil/stigs/downloads/
+levels:
+    -   id: high
+    -   id: medium
+    -   id: low
+controls:
+    -   id: needed_rules
+        levels:
+            - medium
+        rules:
+            - enable_authselect
+            - var_authselect_profile=sssd
+
+    -   id: RHEL-09-211010
+        levels:
+            - high
+        title: RHEL 9 must be a vendor-supported release.
+        rules:
+            - installed_OS_is_vendor_supported
+        status: automated
+
+    -   id: RHEL-09-211015
+        levels:
+            - medium
+        title:
+            RHEL 9 vendor packaged system security patches and updates must be installed
+            and up to date.
+        rules:
+            - security_patches_up_to_date
+        status: automated
+
+    -   id: RHEL-09-211020
+        levels:
+            - medium
+        title:
+            RHEL 9 must display the Standard Mandatory DOD Notice and Consent Banner
+            before granting local or remote access to the system via a command line user logon.
+        rules:
+            - banner_etc_issue
+            - login_banner_text=dod_banners
+        status: automated
+
+    -   id: RHEL-09-211025
+        levels:
+            - medium
+        title: RHEL 9 must implement the Endpoint Security for Linux Threat Prevention tool.
+        rules:
+            - package_mcafeetp_installed
+            - agent_mfetpd_running
+        status: automated
+
+    -   id: RHEL-09-211030
+        levels:
+            - medium
+        title:
+            The graphical display manager must not be the default target on RHEL 9 unless
+            approved.
+        rules:
+            - xwindows_runlevel_target
+        status: automated
+
+    -   id: RHEL-09-211035
+        levels:
+            - low
+        title:
+            RHEL 9 must enable the hardware random number generator entropy gatherer
+            service.
+        rules:
+            - service_rngd_enabled
+        status: automated
```

Comment:
Sounds like the service_rngd_enabled must be completely purged out from the RHEL9 STIG: https://github.com/ComplianceAsCode/content/pull/10153

If that is really true, then we have to request this back to DISA.